contrib/mod_tls.c

```diff
@@ -11984,7 +11984,10 @@ MODRET set_tlsciphersuite(cmd_rec *cmd) {
   c = add_config_param(cmd->argv[0], 1, NULL);
 
   /* Make sure that EXPORT ciphers cannot be used, per Bug#4163. */
-  c->argv[0] = pstrcat(c->pool, "!EXPORT:", ciphersuite, NULL);
+  /* This breaks system profiles though, so don't change them.   */
+  if (strncmp(ciphersuite, "PROFILE=", 8)) {
+    c->argv[0] = pstrcat(c->pool, "!EXPORT:", ciphersuite, NULL);
+  }
```

Comment:
On the other hand, only admins who explicitly request `PROFILE` would know to do so; using `PROFILE` for an older OpenSSL version would lead to an error.  So this change might be fine as is.

A nice addition (not for this PR) would be to use `SSL_CTX_set_cipher_list` on the constructed string, in the configuration directive handler, to detect any issues at startup time.  I'll work on that.